app/config/security.yml

```diff
@@ -8,6 +8,23 @@ security:
     encoders:
         Sylius\Component\User\Model\UserInterface: sha512
     firewalls:
+        admin:
+            switch_user: true
+            context: user
+            pattern: /admin/.*
```

Comment:
With Sylius being open source I don't think that would be obscure enough :) IMO user experience is more important. Anyway, if someone really wants to have backend at some weird path, Security component allows to overwrite this specific configuration option (although it won't let to add new firewall options).